docker-compose/mariadb-config/prod/mariadb.cnf

```diff
@@ -0,0 +1,58 @@
+[mariadb]
+
+port                  = 3306
+bind-address          = 0.0.0.0
+
+# Fine tuning
+max_connections         = 300
+connect_timeout         = 5
+wait_timeout            = 600
+max_allowed_packet      = 32M
+thread_cache_size       = 128
+sort_buffer_size        = 128M
+bulk_insert_buffer_size = 128M
+tmp_table_size          = 32M
+max_heap_table_size     = 32M
+skip-name-resolve       = 1
+innodb_log_file_size    = 2G
+
+# Logging
+# warning log-basename is defined later for replication
+# some log option may be ignored.
+# see: https://mariadb.com/kb/en/mysqld-options/#-log-basename
+# log_error = /var/log/mysql/error.log
+slow_query_log
+slow_query_log_file    = /var/log/mysql/mariadb-slow.log
+long_query_time        = 2
+log_slow_verbosity     = query_plan,explain
```

Comment:
= full for the addition notes/warnings - https://mariadb.com/kb/en/server-system-variables/#log_slow_verbosity

Reply:
Given your other comment about log rotation not working for containers, can we take the precaution to set the verbosity to **Full** some weeks past the migration?

Reply:
The additional amount is less that what' going be logged anyway. Either way, its dynamic.